plugins/output-mongodb/src/main/scala/com/stratio/sparkta/plugin/output/mongodb/MongoDbOutput.scala

```diff
@@ -51,19 +51,20 @@ class MongoDbOutput(keyName: String,
 
   override val retrySleep = properties.getString("retrySleep", DefaultRetrySleep).toInt
 
-  override val idAsField = Try(properties.getString("idAsField").toBoolean).getOrElse(false)
```

Comment:
this is out of the scope of this PR.
If we decide to remove this property we also should remove it from the documentation